indra/llinventory/llpermissions.h

```diff
@@ -353,6 +338,11 @@ bool LLPermissions::allowMoveBy(const LLUUID& agent) const
     return allowOperationBy(PERM_MOVE, agent, LLUUID::null);
 }
 
+bool LLPermissions::allowExportBy(const LLUUID& agent) const
+{
+    return agent == mOwner && agent == mCreator;
```

Comment:
Is it fine that applying to an object that you have permissions to edit won't work?

ex: agent A shares permissions with agent B to edit All A's objects and then B tries to apply a material?

Reply:
Yes.... modifiable does not mean exportable.  We don't have a permissions flag for "can export," so keeping the permissions for exporting as restrictive as possible pending feedback from residents.